manifests/backup/backup-s3-br.yaml

```diff
@@ -4,6 +4,8 @@ kind: Backup
 metadata:
   name: demo1-backup-s3
   namespace: test1
+  annotations:
```

Comment:
This annotations should be commented out.

Reply:
done